pages/hold/request/[id].tsx

```diff
@@ -48,15 +47,21 @@ interface HoldRequestPropsType {
 export default function HoldRequestPage({
   discoveryBibResult,
   discoveryItemResult,
+  deliveryLocations = [],
+  patronId,
   isAuthenticated,
 }: HoldRequestPropsType) {
   const metadataTitle = `Item Request | ${SITE_NAME}`
 
   const bib = new Bib(discoveryBibResult)
   const item = new Item(discoveryItemResult, bib)
```

Comment:
They have to be converted here because bib and item sent from getServerSideProps are the raw response from discovery API (full objects can't be passed from the server since they have to be serialized as JSON)